Keas.Sql/dbo/Tables/TeamApiCodes.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE [dbo].[TeamApiCodes]
+(
+	[Id]     INT            IDENTITY (1, 1) NOT NULL,
+	[TeamId] INT            NOT NULL,
+	[ApiCode] UNIQUEIDENTIFIER NOT NULL,
+	CONSTRAINT [FK_TeamApiCode_Teams_TeamId] FOREIGN KEY ([TeamId]) REFERENCES [dbo].[Teams] ([Id]) ON DELETE CASCADE,

```

Comment:
@srkirkland Does this look right?

Reply:
Looks good but it's probably worth putting an index on api code since that's what we are searching on.